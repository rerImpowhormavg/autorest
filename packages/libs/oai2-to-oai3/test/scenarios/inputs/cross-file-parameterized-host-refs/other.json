{
  "parameters": {
    "GlobalEndpoint": {
      "name": "Endpoint",
      "description": "Shared endpoint param.",
      "x-ms-parameter-location": "client",
      "required": true,
      "type": "string",
      "in": "path",
      "x-ms-skip-url-encoding": true,
      "default": "https://api.cognitive.microsoft.com"
    }
  }
}
