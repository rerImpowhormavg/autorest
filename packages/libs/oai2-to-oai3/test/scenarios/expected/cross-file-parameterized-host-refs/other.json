{
  "servers": [],
  "components": {
    "parameters": {
      "GlobalEndpoint": {
        "name": "Endpoint",
        "in": "path",
        "description": "Shared endpoint param.",
        "required": true,
        "x-ms-parameter-location": "client",
        "x-ms-skip-url-encoding": true,
        "schema": {
          "default": "https://api.cognitive.microsoft.com",
          "type": "string"
        }
      }
    }
  }
}